crates/bevy_core_pipeline/src/skybox/mod.rs

```diff
@@ -70,8 +76,32 @@ impl Plugin for SkyboxPlugin {
 /// To do so, use `EnvironmentMapLight` alongside this component.
 ///
 /// See also <https://en.wikipedia.org/wiki/Skybox_(video_games)>.
-#[derive(Component, ExtractComponent, Clone)]
-pub struct Skybox(pub Handle<Image>);
+#[derive(Component, Clone)]
+pub struct Skybox {
+    pub image: Handle<Image>,
+    /// TODO: Docs
+    pub brightness: f32,
```

Comment:
Perhaps: "Scale factor for the skybox image. After multiplying image samples by brightness, the resulting value should be in units of cd/m^2"